src/model.ts

```diff
@@ -94,7 +95,7 @@ export default class EditableDSVModel extends MutableDataModel {
     // if we are getting the last column.
     if (column === model.columnCount('body') - 1) {
       // Check if we are getting any row but the last.
-      if (row < model.rowCount('body') - 1) {
+      if (row <= model.rowCount('body') - 1) {
```

Comment:
I think we should just go with `row < model.rowCount('body')` instead of adding the equals sign. Just for clarity

Reply:
Agreed.